ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -208,6 +220,7 @@ def __init__(
             SessionInformation._from_grpc_v1(info) for info in self._grpc_session_info
         ]
         self._session_cache: Dict[str, object] = {}
+        self._multiplexer_session_cache: Dict[str, object] = {}
```

Comment:
Ok, keeping multiplexer sessions in a separate cache is fine. That ensures that they can only be accessed with multiplexer APIs, not niswitch APIs intended for relay drivers.